FlowCrypt/App/AppContext.swift

```diff
@@ -154,20 +150,22 @@ class AppContextWithUser: AppContext {
         self.authType = authType
         self.user = user
         self.userId = UserId(email: user.email, name: user.name)
-        self.enterpriseServer = try await EnterpriseServerApi(email: user.email)
+        self.enterpriseServer = try EnterpriseServerApi(email: user.email)
         self.clientConfigurationService = ClientConfigurationService(
             server: enterpriseServer,
             local: LocalClientConfiguration(
                 encryptedStorage: encryptedStorage
             )
         )
 
+        var updatedCombinedPassPhraseStorage = combinedPassPhraseStorage
```

Comment:
Let's call it `combinedPassPhraseStorageWithConfiguration`

Reply:
Just changed